view_all_inc.php

```diff
@@ -112,11 +112,16 @@
 			<div class="btn-group pull-left">
 		<?php
 			$t_filter_param = filter_get_temporary_key_param( $t_filter );
-			$t_filter_param = ( empty( $t_filter_param ) ? '' : '?' ) . $t_filter_param;
+			if( empty( $t_filter_param ) ) {
+				$t_summary_link = 'view_all_set.php?summary=1&temporary=y';
+			} else {
+				$t_summary_link = 'summary_page.php?' . $t_filter_param;
+			}
 			# -- Print and Export links --
 			print_small_button( 'print_all_bug_page.php' . $t_filter_param, lang_get( 'print_all_bug_page_link' ) );
 			print_small_button( 'csv_export.php' . $t_filter_param, lang_get( 'csv_export' ) );
 			print_small_button( 'excel_xml_export.php' . $t_filter_param, lang_get( 'excel_export' ) );
+			print_small_button( $t_summary_link, lang_get( 'summary_link' ) );
```

Comment:
Similar to `layout_print_sidebar` don't offer the button without having `view_summary_threshold` access rights.

Reply:
thanks, fixed